packages/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -102,7 +102,7 @@ const useRootStyles = makeStyles({
     width: '100%',
   },
   circular: theme => ({
-    borderRadius: theme.global.borderRadius.circular,
+    borderRadius: '50000px',
```

Comment:
That's a lot of pixels.  Don't you want 50%? Maybe this is a CSS trick I haven't seen before.

Reply:
We don't want 50% because it makes oval shaped buttons instead of pill-shaped when they are not icon only.

I actually just had a conversation with design and we are changing the value of the token from 50% to 10000px. I'll make that a separate PR.